README.md

```diff
@@ -25,7 +29,95 @@ If neither credentials file and env vars are found Swing will fail.
 To create your access keys see the [official AWS documentation][aws-credentials-docs].
 To know more about the environment variables see [this other documentation][aws-env-vars-docs].
 
-# Necessary AWS IAM permissions
+## Configuration Examples
+
+### Using AWS Client
+
+Install aws-cli using your favourite package manager or windows installer and Run
+
+```
+$ aws configure
+AWS Access Key ID [None]: my-aws-key
+AWS Secret Access Key [None]: my-aws-secret-key
+Default region name [None]: 
+Default output format [None]:
+```
+
+### Using Environment Variables
+
+***Linux***
+
+```
+esport AWS_ACCESS_KEY_ID=my-aws-key 
+export AWS_SECRET_ACCESS_KEY=my-aws-secret-key 
+```
+
+For a persistent configuration copy above lines in your
+
+```~/.bashrc``` or ```~/.zshrc``` depending on your shell 
+
+***Windows***
+
+TODO
+
+# Users
+
+## Quickstart
+
+### Existing projects using swing
+
+On Existing projects it is supposed to have swing.csv somewhere in the code. if it's in the same dir swing is you'll just run
+
+```
+$ swing -d -s pathto/swing.csv
+```
+
+### New projects 
+
+Just start uploading your files on an existing S3 bucket. You must have access to the bucket specified with option `-bucket` (Check the following Usage Section for details)
+
+## Usage
+
+Swing accepts these arguments:
+
+  * `-h` or `-help` prints the help
+  * `-r` or `-region` AWS region in which the bucket is stored
+  * `-b` or `-bucket` name of the bucket to use
+  * `-s` or `-swing-file` specifies a custom CSV file used to write files list on upload and read them on download, defaults to `swing.csv`
+  * `-d` or `-download` starts download of files found in specified swing file
+  * `-h` or `-help` shows the help text
```

Comment:
Same as above, aren't the flags prefixed with `--`?

Reply:
looks like flags in the help have `-` , maybe can be fixed with another PR ?

```
➜  swing git:(devel) ✗ swing --help
Swing version 0.1.3
Usage: swing [-h|--help]  [-y|--yes] [-r|--region "<value>"] -b|--bucket
		"<value>" [-s|--swing-file <file>] 
		[-d|--download] <files...>

	Swing is used to upload and download files from AWS S3.
	After each upload a CSV, the Swing file, is updated to keep track of a file version.
	It's suggested to commit the Swing file in VCS to keep track of files history.
	The Swing file is read when downloading to know which files' version need to be downloaded.

Arguments:
  -b string
    	S3 bucket where to upload or download files (shorthand)
  -bucket string
    	S3 bucket where to upload or download files
  -d	Starts download of files found in specified swing file (shorthand)
  -download
    	Starts download of files found in specified swing file
  -h	Prints help text (shorthand)
  -help
    	Prints help text
  -r string
    	AWS region of bucket (shorthand)
  -region string
    	AWS region of bucket
  -s string
    	CSV file read to know which files to download or written to after files upload (shorthand) (default "swing.csv")
  -swing-file string
    	CSV file read to know which files to download or written to after files upload (default "swing.csv")
  -v	Prints Swing version (shorthand)
  -version
    	Prints Swing version
  -y	Assume yes to all questions (shorthand)
  -yes
    	Assume yes to all questions
```